src/main/java/com/endava/soa_5/basic/Exercise_2.java

```diff
@@ -0,0 +1,25 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_2 {
+    //Write a method that prints if a number is divisible by 3 or 5.
+
+    String message;
+
+
+    String setNumberForResult(int number) {
```

Comment:
You may change your approach by removing message property,  adding number property to your class.
And inside this method you'll just check it's value and will return a message;

Reply:
Done!